docs/backup_and_restore.rst

```diff
@@ -101,89 +88,257 @@ When you are ready to begin the backup, run
+backup, the archive will already be in place).
+
+If you are restoring data onto an existing instance (for example, for data
+recovery purposes), see
+:ref:`Restoring a Backup on an Existing Instance <restore_data>`.
+
+If you are reinstalling SecureDrop and then restoring from a backup (for
+example, for hardware migration, operating system upgrade, or disaster
+recovery purposes), see :ref:`Migrating <migrating>`.
 
-The restore command expects to find a ``.tar.gz`` backup archive in
-``install_files/ansible-base`` under the SecureDrop repository root directory.
-If you are using the same *Admin Workstation* to do a restore from a previous
-backup, it should already be there because it was placed there by the backup
-command. Otherwise, you should copy the backup archive that you wish to restore to
-``install_files/ansible-base``.
+For other data recovery scenarios, see
+:ref:`Additional Information <additional_restore_info>` or `contact Support`_.
 
-.. note:: The backup strategy used for SecureDrop versions prior to 0.3.7
-          created encrypted archives with the extension ``.zip.gpg``.
-          You can safely remove those files once you've created the ``.tar.gz``
-          backup archive described in this guide.
+.. _contact Support: https://securedrop-support.readthedocs.io/en/latest/
 
-Restoring From a Backup File
-''''''''''''''''''''''''''''
+.. _restore_data:
 
-.. important:: This documentation applies to a SecureDrop instance using
-               v2 onion services. If your instance uses v3 onion services,
-               you will need to follow additional steps depending on your
-               specific restore scenario.
+Restoring a Backup on an Existing Instance
+''''''''''''''''''''''''''''''''''''''''''
 
-To perform a restore, you must already have a backup archive. Provide its
-filename in the following command:
+To restore an existing instance to a previous state, run the command:
 
 .. code:: sh
 
-   ./securedrop-admin restore sd-backup-2017-07-22--01-06-25.tar.gz
+   ./securedrop-admin restore sd-backup-2020-07-22--01-06-25.tar.gz
+
+Make sure to replace ``sd-backup-2020-07-22--01-06-25.tar.gz`` with the filename
+for your backup archive.
+
+This command attempts to  restore submissions, source and journalist accounts,
+and configuration details for the onion services used by the web interfaces and
+SSH (if configured) to their state at the time of the backup.
+
+If your backup contains v2 and v3 onion service configuration
+details, but your instance has since been reconfigured to use v3 onion
+services only, the restore command will restore only the v3 onion service
+configuration.
+
+.. note:: You cannot restore the onion service configuration from a v2-only
+  backup onto a v3-only server -  but you can restore submissions and account
+  data using the ``--preserve-tor-config`` argument. See :ref:`Data-only Restores <additional_restore_info>` for more information.
```

Comment:
The preceding two paragraphs have a lot of potential for being confusing, especially in light of the Ubuntu 20.04 migration.  Here's my stab at simplifying them, intended to replace both paragraphs with a single `.. note::`:

> You cannot restore v2 onion service configurations to a v3-only server (this includes any SecureDrop installation based on Ubuntu 20.04). If a restore operation would leave you without a valid onion service configuration, the restore script will fail with an error. In this case, you can still perform a [data-only restore using the `--preserve-tor-config` argument](linky).

Per https://github.com/freedomofpress/securedrop/blob/develop/install_files/ansible-base/roles/restore/files/compare_torrc.py I believe the restore script itself will print reasonably clear instructions, as well.